fe_polars/encoding/target_encoding.py

```diff
@@ -72,7 +72,7 @@ def fit(
             # Compute the count and mean of each group
             agg = x.group_by(feature).agg(
                 [
-                    polars.count().cast(polars.Float64),
```

Comment:
I have opened a new pull request to update the polars version [here](https://github.com/jordandelbar/feature-engineering-polars/pull/38) in order to have the commit messages that respect the conventional commit spec (to correctly determine the version number change and generate the changelog). Can you explain why you change this line? it seems that the tests pass in 1.8.2 with the previous code (I thought you changed it due to a API change in polars from 0.19 -> 1.x.x). Thanks!

Reply:
As referred to the Polars documentation: https://docs.pola.rs/api/python/stable/reference/expressions/api/polars.count.html, `count()` is deprecated and should be replaced with `len()`

Reply:
My bad, you are right! I skimmed through the doc without seeing the warning deprecation. Thank you!